Bio/SeqRecord.py

```diff
@@ -786,31 +829,14 @@ def __len__(self):
         """
         return len(self.seq)
 
-    def __lt__(self, other):
-        """Define the less-than operand (not implemented)."""
-        raise NotImplementedError(_NO_SEQRECORD_COMPARISON)
-
-    def __le__(self, other):
-        """Define the less-than-or-equal-to operand (not implemented)."""
-        raise NotImplementedError(_NO_SEQRECORD_COMPARISON)
-
-    def __eq__(self, other):
-        """Define the equal-to operand (not implemented)."""
-        raise NotImplementedError(_NO_SEQRECORD_COMPARISON)
-
-    def __ne__(self, other):
-        """Define the not-equal-to operand (not implemented)."""
-        raise NotImplementedError(_NO_SEQRECORD_COMPARISON)
-
-    def __gt__(self, other):
-        """Define the greater-than operand (not implemented)."""
-        raise NotImplementedError(_NO_SEQRECORD_COMPARISON)
-
-    def __ge__(self, other):
-        """Define the greater-than-or-equal-to operand (not implemented)."""
-        raise NotImplementedError(_NO_SEQRECORD_COMPARISON)
+    __lt__ = _deleted_seqrecord_comparison_method
```

Comment:
This is more consise, but we lose the docstring.

I would be happier to keep the repetitive stub methods, and annotate them.

Reply:
OK, I've reverted this. I also rebased on current master.